draft-ietf-acme-acme.md

```diff
@@ -3380,7 +3384,13 @@ account holder could take within the scope of ACME:
 
 For this reason, it is RECOMMENDED that account key pairs be used for no other
 purpose besides ACME authentication.  For example, the public key of an account
-key pair SHOULD NOT be included in a certificate.  ACME clients and servers
+key pair SHOULD NOT be included in a certificate.  ACME clients MUST NOT reuse 
+the same account key for multiple accounts, and MUST NOT allow account key 
+roll-over to a previously-used account key.  ACME servers SHOULD reject a 
```

Comment:
I thought a new-account request with a public key already associated with an account is *the* way to retrieve the account URI, given the public key? I think neither SHOULD nor MUST is a good idea because of that.

Reply:
I guess the sentence should be "ACME servers MUST NOT create a new account using an account key already associated with an account on the server.", without mentioning the `new-account` endpoint.